cirq/contrib/quirk/quirk_gate.py

```diff
@@ -116,39 +124,38 @@ def _gate_to_quirk_op(gate: ops.Gate) -> Optional[QuirkOp]:
     return None
 
 
-def x_to_known(gate: ops.XPowGate) -> Optional[QuirkOp]:
-    e = angle_to_exponent_key(gate.exponent)
-    if e is None:
-        return None
-    return QuirkOp('X' + e)
+def xyz_to_known(axis: str, gate: ops.EigenGate) -> QuirkOp:
```

Comment:
Optional nit: "to_known" is weird, because it lacks a noun. How about "xyz_to_quirk_op"? Any ideas for better names for these functions?

(Applies to other functions below, too.)

Reply:
Done.